sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIClientBuilder.java

```diff
@@ -363,10 +364,10 @@ public OpenAIClient buildClient() {
 
     /**
      * OpenAI service can be used by either not setting the endpoint or by setting the endpoint to start with
-     * "https://api.openai.com/"
+     * "https://api.openai.com/" or "https://eu.api.openai.com/".
      */
     private boolean useNonAzureOpenAIService() {
-        return endpoint == null || endpoint.startsWith(OPEN_AI_ENDPOINT);
+        return endpoint == null || endpoint.startsWith(OPEN_AI_ENDPOINT) || endpoint.startsWith(EU_OPEN_AI_ENDPOINT);
```

Comment:
This isn't the most extensible design. Might make more sense to check for whether the endpoint contains the OpenAI domain, but my security best practices are a bit shaky and I'm not sure how easily that could be exploited to direct requests somewhere they're not supposed to go.